internal/api/middleware.go

```diff
@@ -173,7 +173,7 @@ func (a *API) isValidAuthorizedEmail(w http.ResponseWriter, req *http.Request) (
 			}
 		}
 
-		return ctx, badRequestError(ErrorCodeEmailAddressNotAuthorized, "Email address %q cannot be used as it is not authorized", email)
+		return ctx, badRequestError(ErrorCodeEmailAddressNotAuthorized, "Email address %q is not a member of this organizations team.", email)
```

Comment:
the error message shouldn't contain information about the wider Supabase jargon like "members" or "organizations" - we have some folks who self-host the auth service outside of the Supabase ecosystem

Reply:
I'll fix this in the infra.

Reply:
https://github.com/supabase/infrastructure/pull/20244